bin/deferred_work_manager

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+
+require "tom_queue"
+
+module TomQueue
+  class Daemon
+    include LoggingHelper
+
+    def run(argv = ARGV)
+      return unless load_rails!(".")
```

Comment:
Feels like `"."` should be a method argument with a default, same way `argv` is. Or don't bother and just have `bin/deferred_work_manager` hardcoded to use `"."` as the path in `load_rails!`